.github/workflows/verify-generated-code-changes.yml

```diff
@@ -0,0 +1,46 @@
+name: Verify generated code changes
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - 'release/*'
```

Comment:
Is this a "better safe than sorry" approach, ie would all pushes to `release/*` be preceded by a pull request?

Reply:
Ack, thanks for clarifying.